junit-platform-commons/src/main/java/org/junit/platform/commons/util/AnnotationUtils.java

```diff
@@ -228,7 +228,7 @@ else if (candidate.annotationType().equals(containerType)) {
 					// Note: it's not a legitimate container annotation if it doesn't declare
 					// a 'value' attribute that returns an array of the contained annotation type.
 					// Thus we proceed without verifying this assumption.
-					Method method = ReflectionUtils.getMethod(containerType, "value").get();
+					Method method = ReflectionUtils.getMethod(containerType, "value").orElseThrow(AssertionError::new);
```

Comment:
Was my first thought, too. But the existance of the `value` annotation attribute is guaranteed by the JLS (looking for the link), that the simple "get()" was already ok here. The "get()" produced a warning marker in the IDE, though, which is eliminated with the use of "orElseThrow(AssertionError::new)".